### PR TITLE
Convert method names in endpoints to camel case

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #36

### CODE REVIEW

File: src/main/java/org/example/Operations/QueryOperations.java [hunk 1]
```diff
@@ -24,7 +22,7 @@ public QueryOperations(Wrapper client) {
         this.client = client;
         this.baseUrl = client.getBaseUrl() + "/query";
     }
-    public Response execute_genquery(String token, String query, QueryExecuteGenqueryParams params)  {
+    public Response executeGenquery(String token, String query, QueryExecuteGenqueryParams params)  {
```
Comment (korydraughn) [inline src/main/java/org/example/Operations/QueryOperations.java:25]: `executeGenquery` -> `executeGenQuery`

**Review by korydraughn — COMMENTED** [review]
Looks good. Just one comment to resolve.

File: src/main/java/org/example/Operations/QueryOperations.java (hunk 1 above)
Comment (sam-i-am012) [reply src/main/java/org/example/Operations/QueryOperations.java]: Done.

**Comment by sam-i-am012** [comment]
Comment has been resolved!

**Comment by korydraughn** [comment]
When you make changes based on code review comments, avoid squashing them. Instead, push the new commit to the PR. Doing that allows us to see what changed since the last commits. You can come up with a scheme for determining how things should be squashed. For example, the commit that will be squashed later could be worded like so:
```
squash w/ 42. code review - renamed genquery to GenQuery.
```
Then, when all code reviewers agree that things are good, you squash things down. Remember, you can change the order of the commits during a rebase. If you're uncomfortable with doing that, let me know and I'll show you.

**Comment by korydraughn** [comment]
Do the tests still pass?

**Comment by sam-i-am012** [comment]
Ah okay that makes sense, thank you. Yes, the tests do still pass.

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
It looks like these commits could be squashed to one. It's not obvious to me why they're separate. Please explain.

**Comment by sam-i-am012** [comment]
Everything has been squashed down.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**